### ISSUE TITLE
Query: TLD specific PFS Overview

### ISSUE THREAD

**Issue by Lukas1Komarek** [hft-swp2-ws1516/db-api#13]
Hi.

Could you please make a JSON (like we already have http://tls.thejetlag.de:1337/api/v0/pfs/overview) sorted by TLD Domains? This would be used for the World Map. 

``` javascript
[
   { 
      "TLD": "de",
       "pfsSupport": "31354",
       "total": "45314"
   }
   ...
]

```

Thanks for reading this Issue! :eyes:

**Comment by MdaaaaO** [comment]
wip

**Comment by Lukas1Komarek** [comment]
what?

**Comment by zirkler** [comment]
closed by 8e76f79182df99d4cb66f0f64610badf9dcd6ffb
see: http://h2511680.stratoserver.net:1337/api/v0/pfs/overview
